src/web/client/extension.ts

```diff
@@ -270,35 +272,11 @@ export function processWorkspaceStateChanges(context: vscode.ExtensionContext) {
                     const entityInfo: IEntityInfo = {
                         entityId: getFileEntityId(document.uri.fsPath),
                         entityName: getFileEntityName(document.uri.fsPath),
-                        rootWebPageId: WebExtensionContext.entityDataMap.getEntityMap.get(getFileEntityId(document.uri.fsPath))?.rootWebPageId as string
+                        rootWebPageId: getFileRootWebPageId(document.uri.fsPath),
```

Comment:
This can be null value now - are we handling that scenario

Reply:
yeah, we should handle that.